marbles/core/marbles/core/marbles.py

```diff
@@ -253,15 +254,21 @@ def note(self):
 
     @property
     def locals(self):
-        '''Returns a string displaying the public (a.k.a., not internal
```

Comment:
oops this docstring has been wrong for a while

Reply:
whoops